modules/ROOT/pages/import/quick-start.adoc

```diff
@@ -0,0 +1,51 @@
+[[quick-start]]
+:description: This section gives an overview of the Import service.
+= Quick start
+
+The Import service consists of three tabs; *Data sources*, *Graph models*, and *Import jobs*.
+If you haven't previously imported any data, all three are empty, otherwise sources, models, and import jobs are listed here.
+
+[.shadow]
+image::data-source.png[width=800]
+
+== Provide the data
+
+To get started you need to connect to a data source.
+Import supports PostgreSQL, MySQL, SQL Server, as well as locally hosted flat files.
+
+[.shadow]
+image::sources.png[width=400]
+
+For SQL-files, you need to configure the data source, add user credentials for the SQL-database, and give the data source a name.
+If you want to stream local files, you can drag and drop them into the files panel or browse for them.
+
+== Model the data
+
+When you have connected a data source, you are offered to have a model generated based on the data and constraints available in the connected database.
+The quickest way is to accept to have a model generated, but you can draw your own later, see xref:import/modeling.adoc[] for more information.
+
+If you use local files, you can upload a _.json_ file containing your model, if available, or draw your own.
+
+If you generate a model from your SQL data or upload a model locally, the model is already mapped to your files.
```

Comment:
whether the mapping exists already in the json upload depends on whether you did the mapping when exporting the model, so this isn't strictly true

Reply:
Rewritten!